07-routing-gossip.md

```diff
@@ -75,6 +77,9 @@ A recipient node:
     - MAY fail the channel.
   - if it has sent AND received a valid `announcement_signatures` message:
     - SHOULD queue the `channel_announcement` message for its peers.
+  - MAY rely on its counterparty not sending an `announcement_signatures`
```

Comment:
What's this trying to convey? That it's OK for the other side to violate this clause and that you should be prepared for it?

Reply:
You MAY rely on your counterparty to comply with the SHOULD NOT above in order to behave correctly. Otherwise you'd have to not rely on it, as a SHOULD NOT is not sufficient for you to rely on your counterparty following it.